src/DataTable/index.jsx

```diff
@@ -73,19 +73,14 @@ function DataTable({
     // Stringifying the data gives a quick way of checking deep equality
   }, [fetchData, JSON.stringify(instance.state)]);
 
-  const enhancedInstance = useMemo(() => ({
+  const enhancedInstance = {
```

Comment:
lol, the `instance` here is the react table instance right?

Reply:
yep